examples/job/expansions/job-expand.py

```diff
@@ -0,0 +1,46 @@
+#! /usr/bin/python
+# -*- coding: utf-8 -*-
+
+
+params = [
+		{
+			"name": "apple",
+			"args": r"./apple.txt",
+		},
+		{
+			"name": "banana",
+			"args": r"./香蕉.txt",
+		},
+		{
+			"name": "cherry",
+			"args": r"../вишня.doc",
+		}
+	]
+
+for p in params:
+  print (
```

Comment:
I've did slight change to this print line and transformed it into:
```
+    with open("jobs/job-%s.yaml"%p["name"], "w") as f:
+        f.write("""
```
This will create files similarly to what the sed from previous example.